### PR TITLE
Duct Tape - Improvised First Aid/Robolimb Repair!

### PR DESCRIPTION
**Pull request by Software-Pirate-Captain**
I think I'm using Git right this time.

This PR will enable Duct Tape to be used as ghetto first aid and perform minor repairs on robotic limbs. My only real worry with this is that rolls of Duct Tape do not currently have a usage limit like cable coils or regular first aid do, so despite being inferior it's still infinite use. If this is a concern, I can look at adding that functionality and coming back to this.

### CODE REVIEW

**Comment by AnalWerewolf** [comment]
I reckon the regular description is better, if a bit silly. Changing it to directly state that the tape can be used as a first aid tool is a bit silly considering how ghetto it is, not many people would or should know about it. It definitely shouldn't replace regular gauze packs and ointment either.

**Comment by CameronWoof** [comment]
Yeah, this is pretty far out of line. This wholesale replaces gauze and ointment, as it does both jobs and has no usage limit. I definitely don't think this is a good idea in its current state.

**Comment by Software-Pirate-Captain** [comment]
I've made some chages so that applying duct-tape to a wound now only stops it **bleeding**, it will no longer **bandage** or **salve** wounds and thus, will no longer cause them to actually heal any quicker. Nor will it prevent **infection**, it now _only_ stops **bleeding**.

I've a little bit of tweaking to finalise this and I'll have the commit tomorrow. Is this enough to alleviate concerns, or would the usage limit also be a good shout to implent as well, do we think?

**Comment by AnalWerewolf** [comment]
Sounds good to me.

**Comment by AnalWerewolf** [comment]
My reviews still stand, change or close 'em, but as is I don't see this being merged in its current state.

File: code/game/objects/items/weapons/tape.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/tape_roll
 	name = "tape roll"
-	desc = "A roll of sticky tape. Possibly for taping ducks... or was that ducts?"
```
[the remark this answers is not in the document]
Comment (nik707) [reply code/game/objects/items/weapons/tape.dm]: ^

**Comment by kevinz000** [comment]
stale

**Pull request closed without merging**